sos.spec

```diff
@@ -10,7 +10,7 @@ BuildRequires: python3-devel
 BuildRequires: python3-setuptools
 Requires: python3-rpm
 Requires: python3-pexpect
-Requires: python3-setuptools
+Requires: python3-packaging
```

Comment:
Note that in RHEL 9+ and Fedora, this line is redundant, as the dependency is generated, but so is the one for python3-pexpect above, so I decided to keep it consistent.

Reply:
Sadly CentOS 8 and RHEL8 does not seem to have `python3-packaging` package, until I overlooked something..?

Reply:
Not in BaseOS at least. I had no idea you do rebases in RHEL 8. This can be conditionalized, but only in spec, not in the setup.py package metadata.

Reply:
We do sos rebases in RHEL8. Please set this conditional in spec file, if nothing else to let centos-stream-8 CI tests to succeed.